drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -446,6 +446,33 @@ def project_configs
       Hmis::ProjectConfig.all
     end
 
+    field :can_project_accept_referral, Boolean, null: false do
```

Comment:
This query ended up here, rather than on the Project, because any query to the Project fails if the user doesn't have permission to view that project. That doesn't work for this purpose, since we want users to be able to make this query even if they don't have project viewing permission.

I'd be glad to further improve this -- do you think it would rather live on the referring Enrollment, for example?